oozie-to-airflow/tests/mappers/test_subworkflow_mapper.py

```diff
@@ -67,17 +67,7 @@ def test_create_mapper_jinja(self, parse_els):
         with suppress(OSError):
             os.remove("/tmp/test.test_id.py")
         # When
-        mapper = subworkflow_mapper.SubworkflowMapper(
-            oozie_node=self.subworkflow_node,
-            name="test_id",
-            dag_name="test",
-            input_directory_path=EXAMPLE_SUBWORKFLOW_PATH,
-            output_directory_path="/tmp",
-            action_mapper=ACTION_MAP,
-            control_mapper=CONTROL_MAP,
-            trigger_rule=TriggerRule.DUMMY,
-            params=self.main_params,
-        )
+        mapper = self._get_mapper()
```

Comment:
_get_subworkflow_mapper()

Reply:
But.... why? (Applies to all of the above)
Since we're in a test for a particular mapper, isn't a generic `_get_mapper()` cleaner?

Reply:
Cause ouncan seenunderstand what kind of mapper you look like when you see only part of the file. I love to see exactly what's going on when looking at the treat code without having to know which file I am looking at (usually name of the file is not that visible / obvious) an dyou focus on the method's code only.

Reply:
Ok, you convinced me. I was going for shortness and cohesion among all of the test files, but you're right that it might be better to know which type of mapper is returned without going to the method's implementation or looking at the file name.

I'll change it.

Reply:
Done for all of the above.